webapp/sources/pom.xml

```diff
@@ -365,58 +365,57 @@ limitations under the License.
     <rudder-major-version>7.0</rudder-major-version>
     <rudder-version>7.0.0~alpha1-SNAPSHOT</rudder-version>
 
-    <scala-version>2.13.3</scala-version>
+    <scala-version>2.13.6</scala-version>
     <scala-binary-version>2.13</scala-binary-version>
     <scala-xml-version>1.3.0</scala-xml-version>
-    <lift-version>3.4.2</lift-version>
-    <slf4j-version>1.7.30</slf4j-version>
-    <logback-version>1.2.3</logback-version>
-    <junit-version>4.13.1</junit-version>
-    <jodatime-version>2.10.7</jodatime-version>
+    <lift-version>3.4.3</lift-version>
+    <slf4j-version>1.7.32</slf4j-version>
+    <logback-version>1.2.5</logback-version>
+    <junit-version>4.13.2</junit-version>
+    <jodatime-version>2.10.10</jodatime-version>
     <jodaconvert-version>2.2.1</jodaconvert-version>
-    <commons-io-version>2.8.0</commons-io-version>
-    <commons-lang-version>3.11</commons-lang-version>
+    <commons-io-version>2.11.0</commons-io-version>
+    <commons-lang-version>3.12.0</commons-lang-version>
     <commons-codec-version>1.15</commons-codec-version>
     <commons-fileupload>1.4</commons-fileupload>
-    <spring-version>5.2.15.RELEASE</spring-version>
-    <spring-security-version>5.3.10.RELEASE</spring-security-version>
+    <spring-version>5.3.9</spring-version>
+    <spring-security-version>5.5.1</spring-security-version>
     <jgit-version>5.12.0.202106070339-r</jgit-version>
     <cglib-version>3.3.0</cglib-version>
     <asm-version>5.2</asm-version>
-    <bcpkix-jdk15on-version>1.68</bcpkix-jdk15on-version>
-    <silencer-lib-version>1.7.1</silencer-lib-version>
+    <bcpkix-jdk15on-version>1.69</bcpkix-jdk15on-version>
+    <silencer-lib-version>1.7.5</silencer-lib-version>
     <better-files-version>3.9.1</better-files-version>
-    <sourcecode-version>0.2.1</sourcecode-version>
-    <quicklens-version>1.6.1</quicklens-version>
-    <hikaricp-version>3.4.5</hikaricp-version>
-    <nuprocess-version>2.0.1</nuprocess-version>
-    <postgresql-version>42.2.20</postgresql-version>
-    <json-path-version>2.4.0</json-path-version>
-    <json-smart-version>2.3.1</json-smart-version> <!-- json path dependency -->
+    <sourcecode-version>0.2.7</sourcecode-version>
+    <quicklens-version>1.7.4</quicklens-version>
+    <hikaricp-version>4.0.3</hikaricp-version>
+    <nuprocess-version>2.0.2</nuprocess-version>
```

Comment:
this breaks command executions: https://github.com/brettwooldridge/NuProcess/issues/128